pints/_mcmc/_adaptive_covariance.py

```diff
@@ -14,18 +14,11 @@
 
 class AdaptiveCovarianceMCMC(pints.SingleChainMCMC):
     """
-    Adaptive covariance MCMC, as described in [1, 2].
+    Base class for single chain MCMC methods that adapt a covariance matrix
+    when running, in order to control the acceptance rate.
 
-    Using a covariance matrix, that is tuned so that the acceptance rate of the
-    MCMC steps converges to a user specified value.
-
-    [1] Uncertainty and variability in models of the cardiac action potential:
-    Can we build trustworthy models?
-    Johnstone, Chang, Bardenet, de Boer, Gavaghan, Pathmanathan, Clayton,
-    Mirams (2015) Journal of Molecular and Cellular Cardiology
-
-    [2] An adaptive Metropolis algorithm
-    Heikki Haario, Eero Saksman, and Johanna Tamminen (2001) Bernoulli
+    In all cases ``self._adaptations ^ -eta`` is used to control decay of
```

Comment:
Could we have a bit more explanation here of what an adaptive covariance mcmc method is?